### DISCUSSION TITLE
请教是否可以逐小时对比数据条数增量？

### DISCUSSION THREAD

**Discussion by yyri** [datavane/datavines#478] [Q&A] [answered]
请教是否可以逐小时对比数据条数增量？
即，某个表在整点，对比当时总的记录条数，和上个小时同一时间点的记录条数？

**Comment by zixi0825** [comment, marked as answer]
理论上可行，需要你的表存在一个时间字段，通过两表值比对的方式，实际值是计算当前时间点的数量，期望值是计算一个小时前的数量，这样就可以进行比较了

**Reply by yyri** [reply]
谢谢，我去试一下
